lib/test_prof/recipes/rspec/let_it_be.rb

```diff
@@ -145,7 +210,35 @@ def define_let_it_be_methods(identifier, **modifiers)
       end
       record
     end
+
+    config.register_modifier :freeze do |record, val|
+      # TODO: change this to `if val == false` when TestProf hits 1.0
+      unless val == true
+        TestProf::LetItBe::Stoplist.push(record)
+        next record
+      end
+
+      TestProf::LetItBe::Freezer.deep_freeze(record)
+      record
+    end
   end
```

Comment:
I've added a non-AR version here, like (simplified):
```ruby
else
  config.register_modifier :freeze do |record, val|
    record.freeze if val
  end
end
```
But removed it. We may consider adding it later on.